seedling-ontology.ttl

```diff
@@ -0,0 +1,1897 @@
+@prefix : <http://aida.org/ontologies/DomainOntology/> .
```

Comment:
Perhaps we should use `.../SeedlingOntology` rathet than `.../DomainOntology`, reserving the latter for the real one when it is finalized?

Reply:
Good catch. Wilco.